packages/ui-components/package.json

```diff
@@ -6,11 +6,14 @@
     "type": "git",
     "url": "https://github.com/cockroachdb/ui"
   },
-  "main": "dist/main.js",
+  "source": "src/index.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "types": "dist/types/index.d.ts",
   "scripts": {
-    "build": "npm-run-all build:typescript build:bundle",
+    "build": "npm-run-all build:typescript build:parcel",
     "build:bundle": "webpack --mode production",
```

Comment:
Do we need this anymore?

Reply:
Nope! Gotta remove all the webpack stuff before merging!

Reply:
Gone!